pkg/certificate/types.go

```diff
@@ -96,7 +97,37 @@ type Manager struct {
 // MRCClient is an interface that can watch for changes to the MRC. It is typically backed by a k8s informer.
 type MRCClient interface {
 	List() ([]*v1alpha2.MeshRootCertificate, error)
+	MRCEventBroker
```

Comment:
Does the MRCEventBroker need to exist in addition to the MRCClient or can the watch method be defined here?

Watch(context.Context) (<-chan MRCEvent, error)

Reply:
We'll, in the final version of this, I'll implement the Transform function for the message broker so that it can fulfill this interface if need be. Having a single type makes that easier